contrib/babelfishpg_tsql/sql/sys_function_helpers.sql

```diff
@@ -10360,3 +10360,13 @@ LANGUAGE C IMMUTABLE STRICT;
 
 CREATE OR REPLACE FUNCTION sys.bbf_get_context_info()
 RETURNS sys.VARBINARY(128) AS 'babelfishpg_tsql', 'bbf_get_context_info' LANGUAGE C STABLE;
+
+/*
+ * get_immediate_base_type_of_UDT()
+ * This function returns the Immediate base type for UDT.
+ * Returns NULL if given type is not an UDT
+ */
+CREATE OR REPLACE FUNCTION sys.get_immediate_base_type_of_UDT(OID)
```

Comment:
We should follow the norm of prefixing the helper functions with `bbf_` or `babelfish_`.

Reply:
done